content/docs/programs/requisitions.md

```diff
@@ -73,6 +73,55 @@ The list view has gained some additional columns, showing the program related da
 - **Order type**: the name of the order type ( typically, a normal or emergency order )
 - **Period**: the name of the period selected
 
-And the detail view also has program information shown:
+#### Manual Program Requisitions
+
+You can create a manual program requisition for customers who do not use mSupply
+and those who do (for those who do, this should only be done in exceptional).
+
+See the configuration steps above, and also the [Customer
+tag](https://docs.msupply.org.nz/names:adding_and_editing#tags_tab)
+documentation to find out how to set up programs and add tags to customers.
+
+Click on the `New Requisition` button (top right corner).
+
+![New Requisition](/docs/programs/images/add_requisition_button.png)
+
+If you have programs set up, you'll see a different modal to the usual one:
+
+![New Program Requisition](/docs/programs/images/add_program_requisition.png)
+
+Here, you can select the:
```

Comment:
```suggestion
From the `Program` tab, you can create a Program Requisition by selecting the:
```

Reply:
Feels like the screenshot already implies that...